modules/ossm-remove-cleanup.adoc

```diff
@@ -21,7 +21,7 @@ You can manually remove resources left behind after removing the {ProductName} O
 +
```

Comment:
Though this is not in scope for this PR, could you hyphenate "stand-alone service" to align with IBM Style Guide suggestion?

Reply:
@bobfuru thank you for your review and this suggestion. Resolved.